README.md

```diff
@@ -47,12 +47,16 @@ Coming soon! Readings should appear at least 1 month before the date of discussi
 
 ### Alternatives to Capitalist Structures
 
-Topic description coming soon!
+When we explore current centralized data models, many of our fears and challenges are driven by the power of capitalist incentives: reduction of privacy, control from advertisers, and concentration of data ownership into a few corporations are all justified by capitalism's imperative for delivery of maximum value to shareholders.
```

Comment:
list structure year a bit hard to follow -- is everything after the colon fears // challenges? Seems like there isn't agreement between items

Reply:
should have been a semicolon instead of a colon I think. Yes, the list is of fears & challenges– I think there's grammatical agreement, am I missing something? Feel free to take a crack at it